tests/Unit/PointwiseFunctions/AnalyticSolutions/Test_Errors.cpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/ComputeTags.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/System.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Errors.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugeWave.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+SPECTRE_TEST_CASE("Unit.AnalyticSolutions.Errors",
+                  "[Unit][PointwiseFunctions]") {
+  static constexpr size_t volume_dim = 1;
+  using system = GeneralizedHarmonic::System<volume_dim>;
+  using solution = GeneralizedHarmonic::Solutions::WrappedGr<
+      gr::Solutions::GaugeWave<volume_dim>>;
+  using solution_tag = Tags::AnalyticSolution<solution>;
+
+  using simple_tags =
+      tmpl::list<domain::Tags::Coordinates<volume_dim, Frame::Inertial>,
+                 Tags::Time, system::variables_tag, solution_tag>;
+  using compute_tags = tmpl::list<
+      evolution::Tags::AnalyticCompute<volume_dim, solution_tag,
+                                       solution::tags<DataVector>>,
+      Tags::DifferenceFromAnalyticCompute<system::variables_tag::tags_list,
+                                          solution::tags<DataVector>>>;
```

Comment:
are these two taglists the same or different?  If the same, please test the subset case...

Reply:
I chose this test case in part because they are different :)